test_security/CMakeLists.txt

```diff
@@ -243,8 +237,6 @@ if(BUILD_TESTING)
             "${CMAKE_CURRENT_BINARY_DIR}/test_secure_publisher_subscriber${test_suffix}_$<CONFIG>.py"
             TARGET test_secure_publisher_subscriber${test_suffix}
             APPEND_LIBRARY_DIRS "${append_library_dirs}"
-            # on bionic, Connext needs RTI's OpenSSL binaries to be on the PATH
-            # at runtime as the system version of OpenSSL is not supported
```

Comment:
Went back and forth a bit about whether I thought we should keep some version of the comment here but it's a lot of duplication and I think the comment is better kept in the one place where we set the test path. :+1:

Reply:
That was my feeling as well when making that change :+1: probably should've formilated that when I moved this PR from draft to ready